File: src/test/java/com/back/together02be/support/ControllerTestSupport.java
package com.back.together02be.support;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.webmvc.test.autoconfigure.AutoConfigureMockMvc;
import org.springframework.test.web.servlet.MockMvc;

@AutoConfigureMockMvc
public abstract class ControllerTestSupport extends IntegrationTestSupport {

	@Autowired
	protected MockMvc mockMvc;
}

File: src/test/kotlin/com/back/together02be/support/ControllerTestSupport.kt
package com.back.together02be.support

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.webmvc.test.autoconfigure.AutoConfigureMockMvc
import org.springframework.test.web.servlet.MockMvc

@AutoConfigureMockMvc
abstract class ControllerTestSupport : IntegrationTestSupport() {

    @Autowired
    protected lateinit var mockMvc: MockMvc
}
